lms/djangoapps/courseware/views/views.py

```diff
@@ -283,8 +283,9 @@ def course_info(request, course_id):
             # redirect to the dashboard page.
             if isinstance(access_response, StartDateError):
                 start_date = strftime_localized(course.start, 'SHORT_DATE')
```

Comment:
We don't want to change `strftime_localized` to encode its output because this is only a problem when creating query parameters.